### PR TITLE
'Transaction details' page update

### PR DESCRIPTION
**Pull request by TeslaSesla**
Fixed according to the screenshot here: https://github.com/sugarchain-project/translation_android_wallet/issues/38

### CODE REVIEW

**Review by decryp2kanon — APPROVED** [review]
thanks!

File: ru.json [hunk 1]
```diff
@@ -141,11 +141,11 @@
 	"transactionDetails.sendText": "Отправлено",
 	"transactionDetails.receivedText": "Получено",
 	"transactionDetails.dateTooltipText": "Дата",
-	"transactionDetails.confirmationsTooltipText": "Подтверждение",
+	"transactionDetails.confirmationsTooltipText": "Подтверждения",
 	"transactionDetails.txidTooltipText": "Txid",
 	"transactionDetails.fromTooltipText": "От",
-	"transactionDetails.toTooltipText": "До",
-	"transactionDetails.explorerButton": "Блокчейн обозреватель",
+	"transactionDetails.toTooltipText": "Получатель",
```
Comment (decryp2kanon) [inline ru.json]: Oleh Basystiuk, [07.02.20 09:09]
147 row

Oleh Basystiuk, [07.02.20 09:09]
bit long

Oleh Basystiuk, [07.02.20 09:09]
will look ugly

**Review by decryp2kanon — APPROVED** [review]
thanks @obasys

**Review by obasys — APPROVED** [review]
thanks, but don't change "До"  in case we will have different displaying of words, like "От" but "Получатель" looks not really good

**Pull request merged**